features/admin.administrators.v1/pages/administrators.tsx

```diff
@@ -214,6 +214,10 @@ const CollaboratorsPage: FunctionComponent<CollaboratorsPageInterface> = (
         startIndex: 0,
         totalResults: 0
     });
+    const enaleSCIM2RoleAPIV3: boolean = useSelector(
+        (state: AppState) => state.config.ui.enableScim2RolesV3Api
+    );
+    const getRole: (roleId: string) => Promise<AxiosResponse> = enaleSCIM2RoleAPIV3 ? getRoleByIdV3 : getRoleById;
```

Comment:
shall we rename as follows to preserve consistency?

```suggestion
    const getRoleFunction: (roleId: string) => Promise<AxiosResponse> = enaleSCIM2RoleAPIV3 ? getRoleByIdV3 : getRoleById;
```

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/8184/commits/364f88e7a85aa677d870962f791af4ab219be515